packages/meditrak-app/app/api/TupaiaApi.js

```diff
@@ -148,7 +148,7 @@ export class TupaiaApi {
     const response = await this.post(
       CHANGE_USER_PASSWORD_ENDPOINT,
       null,
-      JSON.stringify({ oldPassword, newPassword, newPasswordConfirm }),
+      JSON.stringify({ oldPassword, password: newPassword, passwordConfirm: newPasswordConfirm }),
```

Comment:
The issue was just because the parameter names don't match with the ones in meditrak-server endpoint. Another way to fix this would be to change the parameter names in meditrak-server to match these parameters in meditrak-app. That way we can release the fix tomorrow, but it would require changing other places that use this endpoint too (tupaia, psss, admin-panel,...). So thats why I went with changing it here

Reply:
Can we go with accepting both alternatives in meditrak-server? It's just that even if we change it in the new version of the app, users with older versions of the app will still be unable to change their password.

Reply:
that's a good point!